eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -495,7 +497,7 @@ final case class DATA_WAIT_FOR_DUAL_FUNDING_CONFIRMED(commitments: Commitments,
                                                       previousFundingTxs: List[DualFundingTx],
                                                       waitingSince: BlockHeight, // how long have we been waiting for a funding tx to confirm
                                                       lastChecked: BlockHeight, // last time we checked if the channel was double-spent
-                                                      rbfAttempt: Option[typed.ActorRef[InteractiveTxBuilder.Command]],
+                                                      rbfAttempt: Option[Either[CMD_BUMP_FUNDING_FEE, typed.ActorRef[InteractiveTxBuilder.Command]]],
```

Comment:
Maybe:
```scala
sealed trait RbfStatus
object RbfStatus {
  case object NoRbf extends RbfStatus
  case class RbfRequested(cmd: CMD_BUMP_FUNDING_FEE) extends RbfStatus
  case class RbfInProgress(rbf: typed.ActorRef[InteractiveTxBuilder.Command]) extends RbfStatus
}
```

Reply:
I'll try that.

Reply:
This is a nice clean-up, done in https://github.com/ACINQ/eclair/pull/2275/commits/439c45c9f90fd404359dfe8182157b82dd6fa2c9